docs/src/pages/docs/functions/helpers.md

```diff
@@ -662,6 +662,42 @@ safeMax(data)
 mconcat :: Monoid m, Foldable f => m -> f a -> m a
 ```
 
+
+```javascript
+import mconcat from 'crocks/helpers/mconcat'
+
+import Pred from 'crocks/Pred'
+import compose from 'crocks/helpers/compose'
+import getPropOr from 'crocks/helpers/getPropOr'
+import flip from 'crocks/combinators/flip'
+
+// Record that needs validation
+const person = { name: 'Maisie', age: 22, referral: 'ANON' }
+
+// Helpers -- predicate functions
+const isOfAge = age => age >= 21
+const isReferred = who => who === 'FRIEND'
+
+// Helpers -- point-free functions
+const defProp = flip(getPropOr)
+const getReferral = defProp('referral', 'ANON')
```

Comment:
This will be a little harder for sigs, also lets keep it pretty simple with just using the pure getPropOr, no need to `flip` :)

And because we defined the `Person` type above we can do the following type signature here
```
// getReferral :: Person -> Maybe String
```